hw03_frequency_analysis/go.mod

```
@@ -1,6 +1,6 @@
-module github.com/fixme_my_friend/hw03_frequency_analysis
+module github.com/KootuckB/hw03_frequency_analysis
```

```suggestion
module github.com/Kootuck/otus-go-hw/hw03_frequency_analysis
```
Следует указать полный путь до модуля